mmdet/models/dense_heads/detr_head.py

```diff
@@ -223,13 +224,13 @@ def loss_by_feat(
         assert batch_gt_instances_ignore is None, \
             f'{self.__class__.__name__} only supports ' \
             'for batch_gt_instances_ignore setting to None.'
-
-        losses_cls, losses_bbox, losses_iou = multi_apply(
-            self.loss_by_feat_single,
-            all_layers_cls_scores,
-            all_layers_bbox_preds,
-            batch_gt_instances=batch_gt_instances,
-            batch_img_metas=batch_img_metas)
+        with amp.autocast(enabled=False):
+            losses_cls, losses_bbox, losses_iou = multi_apply(
```

Comment:
Don't we need to find out exactly which operators are not suitable for fp16? Setting the entire loss to fp32 shouldn't be efficient enough？